go/genkit/flow.go

```diff
@@ -430,21 +439,21 @@ func (f *Flow[In, Out, Stream]) runJSON(ctx context.Context, authHeader string,
 }
 
 // provideAuthContext provides auth context for the given auth header if flow auth is configured.
-func (f *Flow[In, Out, Stream]) provideAuthContext(ctx context.Context, authHeader string) (map[string]any, error) {
+func (f *Flow[In, Out, Stream]) provideAuthContext(ctx context.Context, authHeader string) (context.Context, error) {
 	if f.auth != nil {
-		authContext, err := f.auth.ProvideAuthContext(ctx, authHeader)
+		newCtx, err := f.auth.ProvideAuthContext(ctx, authHeader)
 		if err != nil {
-			return nil, fmt.Errorf("unauthorized: %w", err)
+			return ctx, fmt.Errorf("unauthorized: %w", err)
```

Comment:
It's very rare to return a valid value with an error. You should return nil as before.

Reply:
Oops, agree, didn't meant to change it.